### DISCUSSION TITLE
Error loading archive

### DISCUSSION THREAD

**Discussion by zaw007** [Difegue/LANraragi#702] [Q&A] [answered]
**LRR Version and OS**
I use Synology Docker and use Ichaival APP.
LANraragi v0.8.6
Ichaival v1.3.6

**Bug Details**
When reading, it often occurs that only the first few files can be viewed, and the latter files cannot be loaded, so the temporary folder must be emptied.
I found that when this happened, the archives were all decompressed, but the decompressed files were incomplete or corrupted, and checking the log found a line of errors.

**Matching Logs**
Use of uninitialized value in string eq at /root/lanraragi/script/ ../lib/LANraragi/Controller/Api/Archive.pm line 113.

**Screenshots**

**Comment by Hexadecimalism** [comment]
What archive type are you using? cbr/rar or cbz/zip?

**Comment by zaw007** [comment]
> What archive type are you using? cbr/rar or cbz/zip?

rar
compression settings: standard mode,add 5% recovery record.

**Comment by Hexadecimalism** [comment, marked as answer]
> > What archive type are you using? cbr/rar or cbz/zip?
> 
> rar compression settings: standard mode,add 5% recovery record.

I've run into the same issue in the past. Try using zip and see if gives you the same issue.

**Comment by zaw007** [comment]
> > > What archive type are you using? cbr/rar or cbz/zip?
> > 
> > 
> > rar compression settings: standard mode,add 5% recovery record.
> 
> I've run into the same issue in the past. Try using zip and see if gives you the same issue.

Thanks, i'll try it.

**Reply by Difegue** [reply]
👋 Did that fix it? I've marked @Hexadecimalism 's post as the answer in the meantime since .rar support can be fiddly in LRR if you're using too recent a compression algorithm; This is a libarchive limitation I can't really do anything about!

**Reply by zaw007** [reply]
It has been solved, and it is used normally after using zip instead.
